index.ts

```diff
@@ -42,6 +42,13 @@ export {
     automationClientInstance,
 } from "./lib/globals";
 import * as GraphQL from "./lib/graph/graphQL";
+import * as validationPatterns from "./lib/operations/common/params/validationPatterns";
+import * as editModes from "./lib/operations/edit/editModes";
+import * as parseUtils from "./lib/project/util/parseUtils";
+import * as projectUtils from "./lib/project/util/projectUtils";
+import * as secured from "./lib/secured";
+import * as astUtils from "./lib/tree/ast/astUtils";
```

Comment:
Why move all these? And if you are going to break the alphabetical order, why not put them all at the top or bottom?